default.py

```diff
@@ -28,7 +28,7 @@
     (control.lang(30008), "settings", ''),
 ]
 
-_BROWSER = WonderfulSubsBrowser()
+_BROWSER = WonderfulSubsBrowser(control.getSetting('baseflavor'))
```

Comment:
_FALVOR = control.getSetting('baseflavor');
then use it twice.. if we need i as callback, better make a function _get_flavor() then read from setting twice,
the day we decide to rename the key you want to do it only once ;)